sdk/eventhub/Azure.Messaging.EventHubs/src/EventData.cs

```diff
@@ -96,6 +96,16 @@ public override string ContentType
             }
         }
 
+        /// <summary>
+        /// Hidden property that translates the <see cref="MessageWithMetadata.ContentType"/> property to/from a string.
+        /// </summary>
```

Comment:
```suggestion
        /// <summary>
        ///   Hidden property that translates the <see cref="MessageWithMetadata.ContentType"/> property to/from a string.
        /// </summary>
        ///
```

nit:  formatting for consistency with surrounding code.